sdk/schemaregistry/azure-data-schemaregistry/swagger/swagger.yaml

```diff
@@ -0,0 +1,378 @@
+openapi: 3.0.0
```

Comment:
Should the swagger be merged into `azure-sdk-for-java`? I thought this was meant to go into `azure-rest-api-specs`.

Reply:
@srnagar

Reply:
Swagger will go into azure-rest-api-specs once the swagger review board approves it. Since this swagger has not yet been reviewed, I am fine with having it in this repo until then.

@arerlend Can you checkin the json version of swagger instead of yaml